packages/flutter_tools/test/general.shard/ios/devices_test.dart

```diff
@@ -176,60 +176,6 @@ void main() {
       });
     }
 
-    group('ios-deploy wrappers', () {
```

Comment:
Already tested in https://github.com/flutter/flutter/blob/master/packages/flutter_tools/test/general.shard/ios/ios_device_install_test.dart#L79